layerInfo.json

```diff
@@ -166,7 +166,6 @@
 			},
 			"Azure-Examples": {
 				"packages": [
-					"@fluid-example/presence-tracker",
```

Comment:
not sure why presence-tracker is part of Azure-Examples?

To address the failing layer-check error, I have removed this